Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -188,6 +208,31 @@ private void InitializeInternal()
             dictationEventData = new DictationEventData(EventSystem.current);
 
 #endif // UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+
+            if (MixedRealityManager.Instance.ActiveProfile.IsSpeechCommandsEnabled)
+            {
+                SpeechInputSource = new SpeechInputSource(MixedRealityManager.Instance.ActiveProfile.SpeechCommandsProfile.SpeechCommands);
+            }
+
+            DictationInputSource = new DictationInputSource();
+            TouchscreenInputSource = new TouchscreenInputSource();
+            GestureInputSource = new GestureInputSource(MixedRealityManager.Instance.ActiveProfile != null && MixedRealityManager.Instance.ActiveProfile.InputSourceOptions.UseRailsNavigation);
```

Comment:
Why is rails navigation checked here?

Reply:
We're passing in the value assigned to it.

Reply:
i see now